otx/algorithms/action/adapters/mmaction/task.py

```diff
@@ -0,0 +1,525 @@
+            videos_per_gpu=videos_per_gpu,
+            workers_per_gpu=cfg.data.test_dataloader.get("workers_per_gpu", 0),
+            num_gpus=len(cfg.gpu_ids),
+            dist=cfg.distributed,
+            seed=cfg.get("seed", None),
+            shuffle=False,
+        )
+
+        # Target classes
+        if "task_adapt" in cfg:
+            target_classes = cfg.task_adapt.final
+            if len(target_classes) < 1:
+                raise KeyError(
+                    f"target_classes={target_classes} is empty check the metadata from model ckpt or recipe "
+                    "configuration"
+                )
+        else:
+            target_classes = mm_dataset.CLASSES
+
+        # Model
+        model = self.build_model(cfg, fp16=cfg.get("fp16", False))
+        model.CLASSES = target_classes
+        model.eval()
+        model = build_data_parallel(model, cfg, distributed=False)
+
+        # InferenceProgressCallback (Time Monitor enable into Infer task)
+        time_monitor = None
+        if cfg.get("custom_hooks", None):
+            time_monitor = [hook.time_monitor for hook in cfg.custom_hooks if hook.type == "OTXProgressHook"]
+            time_monitor = time_monitor[0] if time_monitor else None
+        if time_monitor is not None:
+
+            # pylint: disable=unused-argument
+            def pre_hook(module, inp):
+                time_monitor.on_test_batch_begin(None, None)
+
+            def hook(module, inp, outp):
+                time_monitor.on_test_batch_end(None, None)
+
+            model.register_forward_pre_hook(pre_hook)
+            model.register_forward_hook(hook)
+
+        eval_predictions = []
+        feature_vectors = []
+        saliency_maps = []
+
+        def dump_features_hook():
+            raise NotImplementedError("get_feature_vector function for mmaction is not implemented")
+
+        # pylint: disable=unused-argument
+        def dummy_dump_features_hook(model, inp, out):
+            feature_vectors.append(None)
+
+        def dump_saliency_hook():
+            raise NotImplementedError("get_saliency_map for mmaction is not implemented")
+
+        # pylint: disable=unused-argument
+        def dummy_dump_saliency_hook(model, inp, out):
+            saliency_maps.append(None)
+
+        feature_vector_hook = dump_features_hook if dump_features else dummy_dump_features_hook
+        saliency_map_hook = dump_saliency_hook if dump_saliency_map else dummy_dump_saliency_hook
+
+        prog_bar = ProgressBar(len(dataloader))
+        with model.module.backbone.register_forward_hook(feature_vector_hook):
+            with model.module.backbone.register_forward_hook(saliency_map_hook):
+                for data in dataloader:
+                    with torch.no_grad():
+                        result = model(return_loss=False, **data)
+                    eval_predictions.extend(result)
+                    for _ in range(len(data)):
+                        prog_bar.update()
+
```

Comment:
```suggestion
prog_bar.file.write("\n")
```
I don't konw certainly, but I saw this code exists after progress bar update. (e.g. otx/algorithms/common/adapters/mmcv/hooks/eval_hook.py)
how about to add it?